src/board/system76/common/battery.c

```diff
@@ -13,6 +14,18 @@ uint16_t battery_charger_input_current = CHARGER_INPUT_CURRENT;
 #define BATTERY_START_DEFAULT 0
 #define BATTERY_END_DEFAULT 100
 
+// Flash address to save charging thresholds to
+static const uint32_t BAT_CFG_ADDR = CONFIG_EC_FLASH_SIZE - (2 * 1024);
```

Comment:
The dynamic keymap has a hardcoded location of `0x1FC00`. I'd recommend to just use a hardcoded value of `0x1F800` here so they live next to each other regardless of the flash size, or adjust the dynamic keymap location to be CONFIG_EC_FLASH_SIZE - 1024;